bndtools.core/_plugin.xml

```diff
@@ -370,6 +376,12 @@
             schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
             sequence="M1+M2+F"
       />
+      
+	<key sequence="Ctrl+7"
```

Comment:
Hi @gamerson, I'm not sure how this snuck into my commit, this was for a different issue I was looking at. I will revise the commit and remove this.